config/rwahs/components/configuration

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# config/rwahs/components/config - Configuration component configuration for integration script.
+#
+
+set -e
+set -u
+
+component='config'
```

Comment:
These are more or less just here so that the file isn't completely empty, I'm pretty sure this particular value won't be needed, although it might be useful in terms of centralising the output messages.  But yes, it should be `configuration` and also the comment at the top, which doesn't match the filename.